ganga/GangaCore/GPIDev/Lib/Registry/RegistrySlice.py

```diff
@@ -479,6 +479,8 @@ def _display(self, interactive=0):
                         #pass
                         if item == "fqid":
                             vals.append(self._get_display_value(obj, item))
+                        elif item == 'subjob status':
```

Comment:
Is this exception still needed?

Reply:
I think so - Would not the `self.subjobs` end up loading the job? With this check that won't happen for old jobs.

Reply:
Ah, yes, you're probably correct. It will load the master job before the subjob data is loaded.
Can't think of a nicer solution that just returning `----` in that case for unloaded jobs. Actually that's also a nice way by eye of seeing which master jobs have been loaded into memory.